Example/HostingExample/ViewController.swift

```diff
@@ -6,7 +6,7 @@
 //
 
 import UIKit
-#if !OEPNSWIFTUI
+#if !OPENSWIFTUI
 import SwiftUI
 #else
 import OpenSwiftUI
```

Comment:
Thanks for the PR.

Would you mind also updating the order here in this PR?

```
#if OPENSWIFTUI
import OpenSwiftUI
#else
import SwiftUI
#endif
```

Reply:
Sure thing!